### ISSUE TITLE
당뇨

### ISSUE THREAD

**Issue by minyu4759-arch** [minyu4759-arch/blank-app#2]
import streamlit as st

import pandas as pd

from sklearn.linear_model import LogisticRegression

# 제목

st.title("🩺 당뇨 예측 웹앱")

# 데이터 불러오기

df = pd.read_csv("diabetes.csv")

X = df.drop("Outcome", axis=1)

y = df["Outcome"]

# 모델 학습

model = LogisticRegression(max_iter=1000)

model.fit(X, y)

st.header("환자 정보 입력")

# 입력값 받기

pregnancies = st.number_input("임신 횟수", 0, 20)

glucose = st.number_input("혈당", 0, 200)

bloodpressure = st.number_input("혈압", 0, 150)

skinthickness = st.number_input("피부 두께", 0, 100)

insulin = st.number_input("인슐린", 0, 900)

bmi = st.number_input("BMI", 0.0, 50.0)

dpf = st.number_input("당뇨 가족력", 0.0, 2.5)

age = st.number_input("나이", 0, 120)

# 예측 버튼

if st.button("예측하기"):

    input_data = [[pregnancies, glucose, bloodpressure, skinthickness,

                   insulin, bmi, dpf, age]]

    prediction = model.predict(input_data)

    if prediction[0] == 1:

        st.error

    else:

        st.success
